### PR TITLE
[ART-6099] Add job to ingest weekly kernel

### PR DESCRIPTION
**Pull request by vfreex**
A new scheduled job `tag-rpms` is added to run `doozer config:tag-rpms` command to ingest weekly candidate kernel as defined in `rpm_deliveries` field of group config.

A slack message will be sent if any rpms are tagged or untagged by this job.

### CODE REVIEW

**Comment by vfreex** [comment]
Requires https://github.com/openshift-eng/doozer/pull/772

Example slack message and job run: https://redhat-internal.slack.com/archives/C048BG69N8H/p1685009365116249

File: pyartcd/pyartcd/cli.py [hunk 1]
```diff
@@ -15,8 +15,6 @@ def click_coroutine(f):
     """ A wrapper to allow to use asyncio with click.
     https://github.com/pallets/click/issues/85
     """
-    f = asyncio.coroutine(f)
```
Comment (vfreex) [inline pyartcd/pyartcd/cli.py:18]: This no longer needed and breaks python 3.11.

**Review by thiagoalessio — APPROVED** [review]
/lgtm

**Review by joepvd — APPROVED** [review]
/lgtm

**Comment by vfreex** [comment]
Rebased.

**Pull request merged**